libs/matdbg/include/matdbg/ShaderInfo.h

```diff
@@ -22,9 +22,15 @@
 #include <filaflat/ChunkContainer.h>
 #include <filaflat/MaterialChunk.h>
 
+#include <private/filament/Variant.h>
+
+#include <utils/bitset.h>
+
 namespace filament {
 namespace matdbg {
 
+using VariantList = utils::bitset<uint64_t, VARIANT_COUNT>;
```

Comment:
Should we share this type between filament and matdbg? It seems it would be easy to change it on one side without updating the other.

Reply:
Yes I struggled to find a good place.  Added a commit that consolidates it to `Variant.h`.